server/index.ts

```diff
@@ -3,8 +3,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { PluginInitializerContext } from '../../../src/core/server';
+import {
+  PluginConfigDescriptor,
+  PluginInitializerContext,
+} from '../../../src/core/server';
 import { FlowFrameworkDashboardsPlugin } from './plugin';
+import { configSchema, ConfigSchema } from '../server/types';
+
+export const config: PluginConfigDescriptor<ConfigSchema> = {
+  schema: configSchema,
+};
```

Comment:
What is the purpose of this export? Is it read by core OSD somewhere?

Reply:
Yes, tyler